ts/utils/notification.ts

```diff
@@ -14,14 +14,26 @@ export enum AuthorizationStatus {
   Provisional = 3
 }
 
+const isAndroid7NougatAPI24OrMore = (version: string | number) =>
```

Comment:
On [Android](https://reactnative-archive-august-2023.netlify.app/docs/0.69/platform#version) `Platform.Version` is `number`. If we come here only if we are on Android, we could use only number. What do you think?

Reply:
Yes, good catch, even more since Javascript handles the `string` to `number` conversion for the `>=` operator automatically so there is really no need to do it manually.